### ISSUE TITLE
Use klima medium for box titles

### ISSUE THREAD

**Issue by wu-lee** [TransitionbyDesign/homemaker#40]
As per @JasLewicki 2/11 feedback document

**Comment by wu-lee** [comment]
The box titles are already Klima Medium, however they are font-weight: bold , so I interpret this just to drop the bold face.

As an aside: the Klima fonts come as a variety of font files, which don't match naturally to the idea in HTML that for every font variant, there can be **bold**, *italic*, and **_bold-italic_**.  Additionally, there are no pre-made font definitions I could find. 

Therefore I had to create my own mapping, in which Klima Medium in bold maps to Klima Heavy: see this commit f7f9e0d54b4ecce820e5fd25

**Comment by JasLewicki** [comment]
Yes please drop the bold. Should look like the red text in the feedback pdf.

**Comment by wu-lee** [comment]
This is implemented now, bold dropped.
